models/topics.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.selectAllTopics = () => {
+    return db.query("SELECT * FROM topics;").then(({ rows }) => {
+        if (rows.length === 0) {
```

Comment:
For the time being we can assume that there will always be topics in the database so a Promise.reject isn't needed here. Good thinking though as this logic will be very relevant when we look at comments later on in the project.